utils/models.py

```
@@ -0,0 +1,178 @@
+        super().__init__(
+            AutoTokenizer.from_pretrained(
+                "premsa/political-bias-prediction-allsides-DeBERTa"
+            ),
+            AutoModelForSequenceClassification.from_pretrained(
+                "premsa/political-bias-prediction-allsides-DeBERTa"
+            ),
+            True,
+            512,
+        )
+
+    def predict(self, article_body: str, truncate_tokens: bool) -> Leaning:
```

Ruff

Missing type annotation for `self` in method

(ANN101)